buildSrc/src/main/groovy/nva-publication-api.java-common-conventions.gradle

```diff
@@ -45,9 +45,9 @@ test {
     //TODO: investigate the usage of a BuildConfig file with BuildConfig plugin
     environment "API_HOST", "localhost"
     environment "API_SCHEME", "https"
+    environment "ID_NAMESPACE", "https://www.example.org"
```

Comment:
trailing / ?

Reply:
That is one of the reasons I made the UriWrapper. To be able to handle slashes at the end:)